polynote-frontend/polynote/cell.js

```diff
@@ -255,6 +255,25 @@ export class CodeCell extends Cell {
 
         this.editor.getModel().cellInstance = this;
 
+        this.editor.onKeyDown((evt) => {
+            const currentPos = this.editor.getPosition();
+            const modelRange = this.editor.getModel().getFullModelRange();
+            if (evt.code === "ArrowUp") {
```

Comment:
Might want to check that the selection is empty (i.e. start position == end position). I.e. if I'm holding shift and press down a bunch of times to select lines, I probably don't want to move to the next cell if I press down once too many. (Maybe I would want that if you could select stuff across multiple cells... but you can't!)